<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleFormWindow</class>
 <widget class="QMainWindow" name="RoleFormWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoleFormWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>782</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>if</string>
    </property>
    <widget class="QComboBox" name="cbFiled">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCondi">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leValue">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>filed</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>condi</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>561</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>value</string>
     </property>
    </widget>
    <zorder>groupBox_5</zorder>
    <zorder>groupBox_4</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>cbFiled</zorder>
    <zorder>cbCondi</zorder>
    <zorder>leValue</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>320</y>
      <width>782</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>review</string>
    </property>
    <widget class="QTextEdit" name="teReview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>761</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>140</y>
      <width>782</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>then</string>
    </property>
    <widget class="QComboBox" name="cbFiledThen">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCondiThen">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>filed</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>condi</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>481</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>value</string>
     </property>
     <widget class="QTextEdit" name="teValueThen">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>441</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>opr</string>
     </property>
     <widget class="QPushButton" name="addThen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_10</zorder>
    <zorder>groupBox_8</zorder>
    <zorder>groupBox_7</zorder>
    <zorder>cbFiledThen</zorder>
    <zorder>cbCondiThen</zorder>
    <zorder>groupBox_9</zorder>
   </widget>
   <zorder>groupBox_6</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_S"/>
  </widget>
  <action name="actionSave_S">
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>Save(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
